webapp/src/ts/providers/translate-utils.provider.ts

```diff
@@ -34,3 +40,23 @@ export class TranslateMessageFormatCompilerProvider extends TranslateCompiler {
     return translations;
   }
 }
+
+@Injectable()
+export class TranslateParserProvider extends TranslateDefaultParser {
+  constructor() {
+    super();
+  }
+
+  interpolate(expr, params) {
+    if (isObjectLike(expr) && expr.fn && isFunction(expr.fn)) {
```

Comment:
` && expr.fn && ` this might be better for performance but does `isFunction` fail if it receives `undefined`?

Reply:
changed.